applications/application_GoldFox.md

```diff
@@ -0,0 +1,39 @@
+# Crust Grant Proposal
+ 
+* **Project Name:** GoldFox 
+* **Team Name:** Dekun Mine
+* **Payment Address:** 0x722ae35004140bd0392813554e5790f5528f3685
+* **Receivable Token:** CRU 
+ 
+> ️ *The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe.*
+ 
+> ️ *When you submit an application for a Grant，You confirm that you have read these Terms and Conditions carefully and that you agree to be bound by them.*
+ 
+### Overview
+ 
+Please provide the following:
+  * A project name: GoldFox
+  * A brief description of your project:
+A set of aggregate mining and lending platforms for CRU based on HECO, BSC and Okchai
+  * An indication of how your project relates to / integrates into Crust:
+In order to solve the issues for the CRUST participants:
+1.Miners or the guarantors that hold CRU, they don't want to sell the coins but want to make profit through lockup.
+2. Miners don't have enough collateral CRU for mining.
+### Project Details 
+1. Our aggregated revenue platform supports multi-chain switching and automatically adds wallet node configuration for users with user-friendly operation.
+2. We aggregate revenue CRU can be lent to our miners to promote ecological cycle.
+GoldFox is based on HECO chain offering loan aggregation, DEFI products can be automatically produced by profit. OFI adopts an open community operation mode, and OFI holders can participate in any proposal and governance of the platform. Below is a brief description of the core products currently in progress.
+1. Vault
+2. Aggregate loan system
+3. Governance
+An overview of the technology stack
+Web3、React、Solidity、Truffle、Go
+ 
+### Ecosystem Fit 
+We support the GPL open source protocol
+ 
+Help us locate your project in the Crust landscape and what problems it tries to solve by answering each of these questions:
+ 
+* Where and how does your project fit into the ecosystem? 
+* What need(s) does your project meet? 
+* Are there any other projects similar to yours in the I
```

Comment:
Please complete your application before submitting your PR

Reply:
You can't remove or change the original file, this is described in Application Process clearly right?

Reply:
Ok